source/includes/Tracking/_clicks.md

```diff
@@ -47,6 +48,7 @@ Engine ID | int | The ID of the specific engine that was used for the query
 
 Key | Represents | Type | Description
 --- | --- | --- | ---
+ls | Log source | string | Definition of the log source and should always be 'searchresult' when recording clicks
```

Comment:
That is not entirely true though. We need to acknowledge the banner statistics situation as well. Something like this perhaps?
```suggestion
ls | Log source | string | Definition of the log source. For tracking regular clicks, it must be set to **searchresult**. For tracking banner clicks, it must be set to **banner**.
```
Removed the quotes in favor of making the text bold instead, as we would otherwise just have to explicitly state to leave out the quotes.